website/i18n/en.json

```diff
@@ -1825,6 +1825,13 @@
     }
   },
   "pages-strings": {
+    "Avolites Titan Manual|no description given": "Avolites Titan Manual",
+    "Official Manual for the Avolites Titan software|no description given": "Official Manual for the Avolites Titan software",
+    "Get Started|no description given": "Get Started",
+    "Bring lighting and video together on one visual canvas|no description given": "Bring lighting and video together on one visual canvas",
+    "Synergy|no description given": "Synergy",
+    "PioneerDJ waveform and BPM sync integration|no description given": "PioneerDJ waveform and BPM sync integration",
+    "Pioneer DJ Integration|no description given": "Pioneer DJ Integration",
```

Comment:
Is it worth adding descriptions to mention where these are used?

Reply:
The description is used to describe the context of the word e.g.
`<translate desc="flower, not verb">Rose</translate>`
Really you shouldn't describe usage because if the phrase were to be used somewhere else that would result in a redundant translation.